.github/workflows/develop.yml

```diff
@@ -149,7 +150,7 @@ jobs:
           cd ckb-vm-test-suite
           cd binary && cargo build --release --target=aarch64-unknown-linux-gnu && cd ..
           cd ..
-          docker run --rm -v `pwd`:/code -t arm64v8/rust bash -c "RISCV=/dummy /code/ckb-vm-test-suite/test.sh --prebuilt-prefix aarch64-unknown-linux-gnu"
+          docker run --rm -v `pwd`:/code -t arm64v8/rust:1.71.1-bookworm bash -c "RISCV=/dummy /code/ckb-vm-test-suite/test.sh --prebuilt-prefix aarch64-unknown-linux-gnu"
```

Comment:
Can we explain a bit why we need to limit to a specific version?

Reply:
When using the latest version, an error now occurs:

```
/code/ckb-vm-test-suite/binary/target/aarch64-unknown-linux-gnu/release/interpreter32: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /code/ckb-vm-test-suite/binary/target/aarch64-unknown-linux-gnu/release/interpreter32)
/code/ckb-vm-test-suite/binary/target/aarch64-unknown-linux-gnu/release/interpreter32: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /code/ckb-vm-test-suite/binary/target/aarch64-unknown-linux-gnu/release/interpreter32)
/code/ckb-vm-test-suite/binary/target/aarch64-unknown-linux-gnu/release/interpreter32: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /code/ckb-vm-test-suite/binary/target/aarch64-unknown-linux-gnu/release/interpreter32)
```

Reply:
My question is this: <https://github.com/nervosnetwork/ckb-vm/actions/runs/7662872609/job/20885033199#step:5:546>

I cannot see why changing it to `1.71.1` first fixes it.

Reply:
[arm64v8/rust:latest](https://hub.docker.com/layers/arm64v8/rust/latest/images/sha256-48e4d3984b3d5ba7b00f452541c7076dacffd19774426be61555b841944a09ed?context=explore)'s architecture is `linux/arm64`
[arm64v8/rust:1.71.1-bookworm](https://hub.docker.com/layers/arm64v8/rust/1.71.1-bookworm/images/sha256-2f1a8ce70c0746044e47691cb8cabb18c5a2e65f88391a020093569d8f2b9e0a?context=explore)'s architecture is `linux/arm64/v8`

I think the difference is here

Reply:
Notice the error line reads:

```
docker: no matching manifest for linux/amd64 in the manifest list entries.
```

It gives an error because **amd64** manifest is missing, not the arm64 manifest is different.

This is making me wonder if there is something weird that is really going on here, and if we should revisit current testing flow

Reply:
Can we try changing this line to:

```
docker run --platform linux/arm64 --rm -v `pwd`:/code -t arm64v8/rust bash -c "RISCV=/dummy /code/ckb-vm-test-suite/test.sh --prebuilt-prefix aarch64-unknown-linux-gnu"
```

And see if the CI works?